Content.Server/Speech/Components/ListenWireAction.cs

```diff
@@ -34,6 +41,7 @@ public override void Initialize()
 
         _wires = EntityManager.System<WiresSystem>();
         _chat = EntityManager.System<ChatSystem>();
+        _prototypeManager = IoCManager.Resolve<IPrototypeManager>();
```

Comment:
Don't these get dependency injected.

Reply:
I assumed they didn't based on the other wire actions, but I have to admit that I didn't actually check!
I'll try it out and see

Reply:
I assumed they didn't based on the other wire actions, but I have to admit that I didn't actually check!
I'll try it out and see.

Reply:
Hm, it appears that they do not. I get null reference exceptions when trying to use the [Dependency] attribute instead.